apps/pages/models.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from django.db import models
+
+from ckeditor.fields import RichTextField
+from mptt.models import MPTTModel, TreeForeignKey
+
+
+class Category(MPTTModel):
```

Comment:
I think your Favro card asked the question - did we need categories for this one?

If we do, then that's okay!

Reply:
https://projects.invisionapp.com/share/58G0EWTWZ6J#/screens/281122376 we do as in can be Information or Features category as per design etc.